woocommerce-gateway-stripe.php

```diff
@@ -197,11 +197,11 @@ public function init() {
 				$this->api                           = new WC_Stripe_Connect_API();
 				$this->connect                       = new WC_Stripe_Connect( $this->api );
 				$this->payment_request_configuration = new WC_Stripe_Payment_Request();
+				$this->account                       = new WC_Stripe_Account( $this->connect, 'WC_Stripe_API' );
 
 				if ( is_admin() ) {
 					require_once dirname( __FILE__ ) . '/includes/admin/class-wc-stripe-admin-notices.php';
 					require_once dirname( __FILE__ ) . '/includes/admin/class-wc-stripe-settings-controller.php';
-					require_once dirname( __FILE__ ) . '/includes/class-wc-stripe-account.php';
```

Comment:
This `require_once` looks redundant to me, because it was already included above.